chunky/src/java/se/llbit/chunky/main/ChunkyOptions.java

```diff
@@ -65,11 +65,24 @@ public static ChunkyOptions getDefaults() {
     clone.sceneDir = sceneDir;
     clone.sceneName = sceneName;
     clone.resourcePacks = new ArrayList<>(resourcePacks);
-    clone.renderThreads = renderThreads;
+    clone.renderOptions.copyState(renderOptions);
     clone.worldDir = worldDir;
     return clone;
   }
 
+  /**
+   * Returns the current state of the render options to by the renderer.
+   * It is not specified whether it is a copy or a view of the current options and
+   * does not guarantee to update on change.
+   * <p>Cast it to {@link ModifiableRenderOptions}, if you have to edit it.
+   * Edits are only safe on startup and might not propagate while a render is in progress.
+   *
+   * @return Options to use to start the render system.
```

Comment:
That sounds flaky to me.

Reply:
Same behaviour as before, but now documented. I wanted to make clear that this should typically only be set on startup (command line args) otherwise requiring a restart (of the render).

Reply:
Great! :clap: